ServerNG.md

```diff
@@ -0,0 +1,155 @@
+Infinispan ServerNG
+====================
+
+Infinispan ServerNG is a reboot of Infinispan's server which addresses the following:
+
+* small codebase with as little duplication of already existing functionality (e.g. configuration)
+* embeddable: the server should allow for easy testability in both single and clustered configurations
+* RESTful admin capabilities
+* Logging using [JBoss Logging logmanager](https://github.com/jboss-logging/jboss-logmanager)
+* Security using [Elytron](https://github.com/wildfly-security/wildfly-elytron)
+
+# Layout
+
+The server is laid out as follows:
+
+* `/bin` scripts
+   * `server.sh` server startup shell script for Unix/Linux
+   * `server.ps1` server startup script for Windows Powershell 
+* `/lib` server jars
+   * `infinispan-server.jar` uber-jar of all dependencies required to run the server.
+* `/server` default server instance folder
+* `/server/log` log files
+* `/server/configuration` configuration files
+   * `infinispan.xml`
```

Comment:
Is there only one configuration or do we have different examples and the default is infinispan.xml which contains a simple clustering?

Reply:
Yes, examples go in docs/examples